lightning/restore/checksum.go

```diff
@@ -10,14 +10,15 @@ import (
 	"time"
 
 	"github.com/google/uuid"
-
 	"github.com/pingcap/br/pkg/checksum"
+	"github.com/pingcap/br/pkg/utils"
```

Comment:
we might use `github.com/cznic/mathutil`, because I remember this path will cause a cycle import after lightning via SQL.

Reply:
It's weird that this dep causes cyclic deps? Since this file allow depends on other br pkgs